seisflows/preprocess/default.py

```diff
@@ -668,6 +674,12 @@ def _apply_normalize(self, st):
 
         return st_out
 
+    def read_residuals(self, residuals_files):
```

Comment:
I think we can expose this as a public function in the script (outside the class definition), so that the Pyatoa class can import and use it. That way we avoid having the same function in two places. 

Also a small docstring would be great, something like
```python
"""
Convenience function to read in text files containing misfit information
written by `quantify_misfit` function

:type residual_files: list of str
:param residual files: list of names of text files written out by each 
misfit quantification procedure
:rtype: np.array
:return: residuals from all files provided in `residual_files`
"""